docs/release/1.16/index.rst

```diff
@@ -0,0 +1,13 @@
+.. ONE documentation master file, created by
+   sphinx-quickstart on Thu May 20 12:56:12 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+1.0
```

Comment:
> I just recognized that there was a problem with updating the release information in our documentation.

@lemmaa Actually, as you said, all of other release note has the same contents about this. It's because those are not from automatic tool but from copy-paste. Could you let me know how to create `index.rst` by related tool?

Reply:
@mhs4670go , unfortunately, it is not created automatically. You have to make it manually. :( 
I will try to make this series of processes more automated in the future when it comes to user document creation.